Avoid trying to run /usr/bin/picolisp since it is probably too
old - just run the java version, since java is needed by the
Termux package builds anyway.

diff -u -r ../picoLisp/src64/mkAsm ./src64/mkAsm
--- ../picoLisp/src64/mkAsm	2011-06-08 12:49:19.000000000 -0400
+++ ./src64/mkAsm	2016-06-21 16:23:47.625814914 -0400
@@ -1,14 +1,6 @@
 #!/bin/sh
 # 08jun11abu
 
-if test -x /usr/bin/picolisp
-then
-   /usr/bin/pil mkAsm.l "$@"
-elif test -x ../bin/picolisp
-then
-   ../pil mkAsm.l "$@"
-else
    ../ersatz/pil mkAsm.l "$@"
-fi
 
 # vi:et:ts=3:sw=3
